curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/5dfa37b9eacea3f48c6300b0.md

```diff
@@ -43,9 +44,24 @@ assert.equal(secondSectionLastElemNode.nodeName, 'UL');
       <h1>CatPhotoApp</h1>
       <section>
         <h2>Cat Photos</h2>
-        <p>Everyone loves <a href="https://cdn.freecodecamp.org/curriculum/cat-photo-app/running-cats.jpg">cute cats</a> online!</p>
-        <p>See more <a target="_blank" href="https://freecatphotoapp.com">cat photos</a> in our gallery.</p>
-        <a href="https://freecatphotoapp.com"><img src="https://cdn.freecodecamp.org/curriculum/cat-photo-app/relaxing-cat.jpg" alt="A cute orange cat lying on its back."></a>
+        <p>
+          Everyone loves
+          <a
+            href="https://cdn.freecodecamp.org/curriculum/cat-photo-app/running-cats.jpg"
+            >cute cats</a
+          >
+          online!
+        </p>
+        <p>
+          See more
+          <a target="_blank" href="https://freecatphotoapp.com">cat photos</a>
+          in our gallery.
+        </p>
+        <a href="https://freecatphotoapp.com"
+          ><img
+            src="https://cdn.freecodecamp.org/curriculum/cat-photo-app/relaxing-cat.jpg"
+            alt="A cute orange cat lying on its back."
+        /></a>
```

Comment:
I think we should either revert this formatting, or change it on all of the steps. It's a little strange to go to one step with this formatting and all the others have the old formatting. I would say to just revert it? What do you think @a2937?

Reply:
Unfortunately yeah I have to revert to the old formatting here. The main idea behind this PR doesn't encompass all the steps.